docs/theming.md

```diff
@@ -1,113 +1,209 @@
-# Theming your Alfresco ADF app
+# Theming an ADF app
+
+The [Material Design](https://material.io/guidelines/material-design/introduction.html)
+language doesn't specify a single color scheme. Instead it uses the concept
+of color **themes** to allow designers some flexibility in their choice of colors.
+
+A theme is a palette based around two main colors: the **primary** color (used widely
+throughout the app) and the **accent** color (used mainly for highlighting and calling
+out specific UI elements). Each of these colors is defined in a number of shades. For
+example, a blue/orange theme could define shades like the following:
+
+![Theme swatches](docassets/images/ThemeSwatches.png)
+
+Each shade is related to a particular purpose or set of purposes within the app. So for
+example, the shade that works best for text isn't necessarily the same shade you would use
+for flat areas of color. Material Design provides a number of
+[standard themes](https://material.io/guidelines/style/color.html#color-themes)
```

Comment:
We should not refer to the material themes but to the one pre built.by us as it was in our previous version